### PR TITLE
(maint) Merge 5.5.x to 6.4.x

### PR DESCRIPTION
**Pull request by kris-bosland**

### CODE REVIEW

File: man/man8/puppet-lookup.8 [hunk 1]
```diff
@@ -1,7 +1,13 @@
 .\" generated with Ronn/v0.7.3
 .\" http://github.com/rtomayko/ronn/tree/0.7.3
 .
+<<<<<<< HEAD
 .TH "LOOKUP" "8" "September 2019" "Puppet, Inc." "Puppet manual"
+||||||| merged common ancestors
+.TH "PUPPET\-LOOKUP" "8" "August 2019" "Puppet, Inc." "Puppet manual"
+=======
+.TH "PUPPET\-LOOKUP" "8" "September 2019" "Puppet, Inc." "Puppet manual"
+>>>>>>> upstream/5.5.x
```
Comment (joshcooper) [inline man/man8/puppet-lookup.8]: Looks like conflict isn't resolved.

Comment (kris-bosland) [reply man/man8/puppet-lookup.8]: I must have forgotten to save the file from my editor.  I redid the mergeup, I think it looks OK now.

**Comment by joshcooper** [comment]
Mentioned in slack that it looks like we need to merge 6.0.x to 6.4.x first, as the commits for 10016 were merged to 6.0.x before the 6.0.x pipeline was removed.

**Comment by joshcooper** [comment]
Closing this as it's superseded by PR #https://github.com/puppetlabs/puppet/pull/7720, and there are no updates requires when merging from 5.5.x to 6.4.x:

```
$  git merge --no-ff upstream/5.5.x
Already up to date.
```

**Pull request closed without merging**